src/script/page/LeftSidebar/panels/Conversations/ConversationTabs/TeamCreation/TeamCreationAccountHeader.tsx

```diff
@@ -35,11 +36,13 @@ interface Props {
 export const TeamCreationAccountHeader = ({onClick}: Props) => {
   return (
     <div css={teamUpgradeAccountBannerContainerCss}>
-      <div>
-        <Icon.InfoIcon />
-        <span className="heading-h4" css={teamUpgradeBannerHeaderCss}>
-          {t('teamUpgradeBannerHeader')}
-        </span>
+      <div css={{flex: '1'}}>
+        <div css={bannerHeaderContainerCss}>
+          <Icon.InfoIcon />
+          <span className="heading-h4" css={teamUpgradeBannerHeaderCss}>
```

Comment:
I have one question: I see three ways of styling here. Do we have a way to go for that kind of thing? Something like a style guide? :) Do we have a preferred way of styling?

Reply:
I can explain but not sure completely, we have defined classes and it is better to consume those, then we have custom css. If the custom css is not lengthy then I prefer to keep it inline otherwise prefer to move in a separate file.